visitors/index.js

```diff
@@ -0,0 +1,159 @@
+var es6ForOf = require('./es6-for-of-visitors');
+var es6ObjectComputedProperty =
+  require('./es6-object-computed-property-visitors');
+var es6ObjectConciseMethod = require('./es6-object-concise-method-visitors');
+var es6ObjectShortNotation = require('./es6-object-short-notation-visitors');
+var es6RestParam = require('./es6-rest-param-visitors');
+var es6Template = require('./es6-template-visitors');
+var es7SpreadProperty = require('./es7-spread-property-visitors');
+var es7TrailingComma = require('./es7-trailing-comma-visitors');
+var reactDisplayName = require('./react-display-name-visitors');
+var reactJSX = require('./react-jsx-visitors');
+var reservedWords = require('./reserved-words-visitors');
+var trailingComma = require('./trailing-comma-visitors');
+var undefinedToVoid0 = require('./undefined-to-void-0-visitors');
+
+// Map from transformName => orderedListOfVisitors.
+var transformVisitors = {
+  'es6-arrow-function': es6ArrowFunction.visitorList,
+  'es6-call-spread': es6CallSpread.visitorList,
+  'es6-class': es6Class.visitorList,
+  'es6-destructuring': es6Destructuring.visitorList,
+  'es6-for-of': es6ForOf.visitorList,
+  'es6-object-computed-property': es6ObjectComputedProperty.visitorList,
+  'es6-object-concise-method': es6ObjectConciseMethod.visitorList,
+  'es6-object-short-notation': es6ObjectShortNotation.visitorList,
+  'es6-rest-param': es6RestParam.visitorList,
+  'es6-template': es6Template.visitorList,
+  'es7-spread-property': es7SpreadProperty.visitorList,
+  'es7-trailing-comma': es7TrailingComma.visitorList,
+  'react-display-name': reactDisplayName.visitorList,
+  'react-jsx': reactJSX.visitorList,
+  'reserved-words': reservedWords.visitorList,
+  'trailing-comma': trailingComma.visitorList,
+  'undefined-to-void-0': undefinedToVoid0.visitorList
+};
+
+
+// Sets of transforms. Useful for quickly building up with simple options.
+var transformSets = {
+  'es6': [
+    'es6-arrow-function',
+    'es6-call-spread',
+    'es6-class',
+    'es6-destructuring',
+    'es6-for-of',
+    'es6-object-computed-property',
+    'es6-object-concise-method',
+    'es6-object-short-notation',
+    'es6-rest-param',
+    'es6-template'
+  ],
+  'es7': [
+    'es7-spread-property',
+    'es7-trailing-comma'
+  ],
+  'react': [
+    'react',
+    'react-display-name'
+  ],
+  'target:es3': [
+    'reserved-words',
+    'trailing-comma'
+  ],
+  'utility': [
+    'undefined-to-void-0'
+  ]
+};
+// harmony is all newer transforms. Define it here so we don't duplicate.
+transformSets.harmony = transformSets.es6.concat(transformSets.es7);
+
+
+// Specifies the order in which each transform should run.
+var transformRunOrder = [
+  'reserved-words',
+  'es6-destructuring',
+  'es6-arrow-function',
+  // needs to be before concice-methods, short-notation, spread-property
+  'es6-object-computed-property',
+  'es6-object-concise-method',
+  'es6-object-short-notation',
+  'es6-classes',
```

Comment:
@zpao Hmm these names aren't matching what you set in `transformVisitors`. `es6-classes` should be `es6-class`